docs/user/adding-glean-to-your-project.md

```diff
@@ -99,16 +99,46 @@ There are [additional parameters](android-build-configuration-options.md) that c
 #### Setting up the dependency
 
 Glean can be consumed through [Carthage](https://github.com/Carthage/Carthage), a dependency manager for macOS and iOS.
-For consuming the latest version of Glean, add the following line to your `Cartfile`::
+For consuming the latest version of Glean, add the following line to your `Cartfile`:
 
 ```
-github "mozilla/glean" "master"
+github "mozilla/glean" "{latest-version}"
 ```
 
+> **Important:** the `{latest-version}` placeholder should be replaced with the version number of the latest Glean SDK release.
+> You can find the version number on the [release page](https://github.com/mozilla/glean/releases/latest).
+
+Then check out and build the new dependency:
+
+```
+carthage bootstrap --platform iOS --color auto --cache-builds
+```
+
+
 #### Integrating with the build system
 
-> **Note:** This is not yet documented, as the exact mechanism is not finalized. See [Bug 1589383](https://bugzilla.mozilla.org/show_bug.cgi?id=1589383) for more information.
+For integration with the build system you can follow the [Carthage Quick Start steps](https://github.com/Carthage/Carthage#quick-start).
+
+1. After building the dependency one drag the built `.framework` binaries from `Carthage/Build/iOS` into your application's Xcode project.
```

Comment:
I actually did this by clicking the `+` icon in the General tab of the Xcode project settings and selecting "Add Other -> Files..." from the dialog that appeared.  This opened a file explorer window where I navigated to the `Carthage/Build/iOS` directory and selected `Glean.framework`

Dragging and dropping would have been nice to know, I think.  I should have suspected Apple would allow such nice things.

Reply:
:D I just copied what Carthage documented.